.github/workflows/build-npm-release.yml

```diff
@@ -0,0 +1,243 @@
+# This workflow will do a clean install of node dependencies, build the source code,
+# run unit tests, perform a SonarCloud scan and publish NPM package ONLY on a tagged release.
+
+# For more information see:
+# https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+# Common FOLIO configurable env:
+#   - YARN_TEST_OPTIONS (options to pass to 'yarn test')
+#   - SQ_ROOT_DIR (root SQ directory to scan relative to top-level directory)
+#   - PUBLISH_MOD_DESCRIPTOR (boolean 'true' or 'false')
+#   - COMPILE_TRANSLATION_FILES (boolean 'true' or 'false')
+
+name: buildNPM Release
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+  workflow_dispatch:
+
+jobs:
+  github-actions-ci:
+    if : ${{ startsWith(github.ref, 'refs/tags/v') }}
+    env:
+      YARN_TEST_OPTIONS: <options to pass to 'yarn test'>
+      SQ_ROOT_DIR: <replace>
+      COMPILE_TRANSLATION_FILES: <boolean set 'true' or 'false'>
+      PUBLISH_MOD_DESCRIPTOR: <publish mod desc to FOLIO registry (true|false)>
+      FOLIO_NPM_REGISTRY: 'https://repository.folio.org/repository/npm-folio/'
+      FOLIO_NPM_REGISTRY_AUTH: '//repository.folio.org/repository/npm-folio/'
+      FOLIO_MD_REGISTRY: 'https://folio-registry.dev.folio.org'
+      NODEJS_VERSION: '12'
```

Comment:
This line is listed as *should be v16* in [FOLIO-3432](https://issues.folio.org/browse/FOLIO-3432)

Reply:
This workflow only triggers during relase. The updated workflow is in build-npm.yml. I am testing right now only with that.